lib/linguist/heuristics.yml

```diff
@@ -505,3 +517,10 @@ named_patterns:
   objectivec: '^\s*(@(interface|class|protocol|property|end|synchronised|selector|implementation)\b|#import\s+.+\.h[">])'
   perl5: '\buse\s+(?:strict\b|v?5\.)'
   perl6: '^\s*(?:use\s+v6\b|\bmodule\b|\b(?:my\s+)?class\b)'
+  plantuml:
+  - '(@startuml|@enduml)'
+  - '(\w|\[).+->\s*(\w|\]).*(:.+)?'
```

Comment:
this line for example - there are more possibilities than just xxx -> yyy, but the vast majority of plantuml files will have something of this form.